### PR TITLE
Move Valuetype Wrapper to the language-server

### PR DESCRIPTION
**Pull request by georg-schwarz**
This PR moves the AstNodeWrapper `Valuetype` to the language-server. 

This refactor will allow building up an *is-subtype* or *is-convertable-to* functionality in the language-server as part of the language instead of part of the execution. Such a similar functionality is currently implemented in the `convertsImplicitlyTo` method.

Changes along the way:
- `isValid` functionality was extracted to a visitor that stays in the execution
- `getStandardRepresentation` functionality was extracted to a separate concept that stays in the execution. Unfortunately, I couldn't solve it as visitor due to the generic return type.

### CODE REVIEW

File: libs/language-server/src/lib/ast/wrappers/value-type/visitors/visitable-valuetype.ts
Comment (felix-oq) [inline libs/language-server/src/lib/ast/wrappers/value-type/visitors/visitable-valuetype.ts:1]: I think this interface can be abolished by moving its method to the `Valuetype` interface

File: libs/execution/src/lib/types/valuetypes/standard-representation.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  BooleanValuetype,
+  DecimalValuetype,
+  IntegerValuetype,
+  PrimitiveType,
+  TextValuetype,
+  Valuetype,
+} from '@jvalue/jayvee-language-server';
+
+export class StandardRepresentationResolver {
+  private readonly DECIMAL_COMMA_SEPARATOR_REGEX = /^[+-]?([0-9]*[,])?[0-9]+$/;
```
Comment (felix-oq) [inline libs/execution/src/lib/types/valuetypes/standard-representation.ts:16]: The same regex is also used in `IsValidVisitor`, so consider extracting it as a global constant.

**Review by felix-oq — APPROVED** [review]
LGTM, only left a few minor remarks

File: libs/execution/src/lib/types/valuetypes/standard-representation.ts (hunk 1 above)
Comment (georg-schwarz) [reply libs/execution/src/lib/types/valuetypes/standard-representation.ts]: Addressed in [c6a0803](https://github.com/jvalue/jayvee/pull/312/commits/c6a08038f19426453a5ad90e66229b861ac2095c)

**Pull request merged**